source/tests/system/nirfmxwlan_driver_api_tests.cpp

```diff
@@ -339,6 +383,805 @@ TEST_F(NiRFmxWLANDriverApiTests, TXPFromExample_FetchData_DataLooksReasonable)
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_symbol_clock_error_mean_response);
+  EXPECT_EQ(0.0, ofdm_mod_acc_fetch_symbol_clock_error_mean_response.symbol_clock_error_mean());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_iq_impairments_response);
+  EXPECT_EQ(0.0, ofdm_mod_acc_fetch_iq_impairments_response.relative_iq_origin_offset_mean());
+  EXPECT_EQ(0.0, ofdm_mod_acc_fetch_iq_impairments_response.iq_gain_imbalance_mean());
+  EXPECT_EQ(0.0, ofdm_mod_acc_fetch_iq_impairments_response.iq_quadrature_error_mean());
+  EXPECT_EQ(0.0, ofdm_mod_acc_fetch_iq_impairments_response.absolute_iq_origin_offset_mean());
+  EXPECT_EQ(0.0, ofdm_mod_acc_fetch_iq_impairments_response.iq_timing_skew_mean());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_ppdu_type_response);
+  EXPECT_EQ(0, ofdm_mod_acc_fetch_ppdu_type_response.ppdu_type());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_mcs_index_response);
+  EXPECT_EQ(0, ofdm_mod_acc_fetch_mcs_index_response.mcs_index());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_guard_interval_type_response);
+  EXPECT_EQ(0, ofdm_mod_acc_fetch_guard_interval_type_response.guard_interval_type());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_lsig_parity_check_status_response);
+  EXPECT_EQ(0, ofdm_mod_acc_fetch_lsig_parity_check_status_response.lsig_parity_check_status());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_sigcrc_status_response);
+  EXPECT_EQ(0, ofdm_mod_acc_fetch_sigcrc_status_response.sig_crc_status());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_sigbcrc_status_response);
+  EXPECT_EQ(0, ofdm_mod_acc_fetch_sigbcrc_status_response.sigb_crc_status());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_pilot_constellation_trace_response);
+  EXPECT_EQ(0.0, ofdm_mod_acc_fetch_pilot_constellation_trace_response.pilot_constellation(0).real());
+  EXPECT_EQ(0.0, ofdm_mod_acc_fetch_pilot_constellation_trace_response.pilot_constellation(0).imaginary());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_data_constellation_trace_response);
+  EXPECT_EQ(0.0, ofdm_mod_acc_fetch_data_constellation_trace_response.data_constellation(0).real());
+  EXPECT_EQ(0.0, ofdm_mod_acc_fetch_data_constellation_trace_response.data_constellation(0).imaginary());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_chain_rmsevm_per_subcarrier_mean_trace_response);
+  EXPECT_EQ(0.0, ofdm_mod_acc_fetch_chain_rmsevm_per_subcarrier_mean_trace_response.x0());
+  EXPECT_EQ(0.0, ofdm_mod_acc_fetch_chain_rmsevm_per_subcarrier_mean_trace_response.dx());
+  EXPECT_EQ(999, ofdm_mod_acc_fetch_chain_rmsevm_per_subcarrier_mean_trace_response.chain_rms_evm_per_subcarrier_mean_size());
+  EXPECT_EQ(999, ofdm_mod_acc_fetch_chain_rmsevm_per_subcarrier_mean_trace_response.chain_rms_evm_per_subcarrier_mean().size());
+  EXPECT_EQ(0.0, ofdm_mod_acc_fetch_chain_rmsevm_per_subcarrier_mean_trace_response.chain_rms_evm_per_subcarrier_mean(0));
+}
+
+// Getting exception on the second internal call of BuildPortString2 (called from BuildPortString on the client):
```

Comment:
These are definitely suspicious.

Reply:
I'll close this thread and keep the convo in the [other one](https://github.com/ni/grpc-device/pull/507#discussion_r798981123) about the investigation on this.